II.write-tests/02-remove-vowels/remove-vowels.js

```diff
@@ -3,17 +3,17 @@ function removeVowels(word) {
 
   var result = [];
 
-  characters.forEach(function(character) {
+  characters.forEach(function (character) {
     if (
-      character === "a" ||
-      character === "o" ||
-      character === "i" ||
-      character === "e" ||
-      character === "u"
+      character.toLowerCase() === "a" ||
+      character.toLowerCase() === "o" ||
+      character.toLowerCase() === "i" ||
+      character.toLowerCase() === "e" ||
+      character.toLowerCase() === "u"
     ) {
-      result.push(character);
+      result.push("");
     } else {
-      result.push("_");
+      result.push(character);
     }
```

Comment:
The idea for those tests, is to add more tests to make your functions work better. So try to do that and we can review them later on